README.md

```diff
@@ -25,10 +28,58 @@
 TODO: Complete this section
 
 ### How to use RepoAuditor
-TODO: Complete this section
 
 <!-- Content below this delimiter will be copied to the generated README.md file. DO NOT REMOVE THIS COMMENT, as it will cause regeneration to fail. -->
 
+Once installed, you can invoke the following to verify if `RepoAuditor` is installed correctly:
+```shell
+repo_auditor --version

```

Comment:
Can you express these instructions in terms of `uv`?

For example `uv run repo_auditor --version`.